qt/scientific_interfaces/Indirect/ContainerSubtraction.h

```diff
@@ -1,8 +1,9 @@
 #ifndef MANTIDQTCUSTOMINTERFACESIDA_CONTAINERSUBTRACTION_H_
 #define MANTIDQTCUSTOMINTERFACESIDA_CONTAINERSUBTRACTION_H_
 
-#include "ui_ContainerSubtraction.h"
 #include "CorrectionsTab.h"
+#include "ui_ContainerSubtraction.h"
+#include <boost/weak_ptr.hpp>
```

Comment:
I do not see the use of boost/weak_ptr in this file, for me it compiles fine without that include

Reply:
This was mistakenly left in!